From ca1a781afbd68697ef2f573557f1f93ec7ab8d3c Mon Sep 17 00:00:00 2001
From: Martin Jansa <Martin.Jansa@gmail.com>
Date: Sun, 29 Apr 2012 00:05:59 +0200
Subject: [PATCH] SConstruct: disable html and man docs building because
 xmlto-native from OE is broken

It will try to load dtd and fail:
| xmlto man gpsctl.xml; mv `basename gpsctl.1` gpsctl.1
| xmlto: /var/lib/jenkins/jobs/shr-core-branches/workspace/shr-core/tmp-eglibc/work/armv4t-oe-linux-gnueabi/gpsd-3.4-r2/gpsd-3.4/gpsctl.xml does not validate (status 3)
| xmlto: Fix document syntax or use --skip-validation option
| I/O error : Attempt to load network entity http://www.oasis-open.org/docbook/xml/4.1.2/docbookx.dtd
| /var/lib/jenkins/jobs/shr-core-branches/workspace/shr-core/tmp-eglibc/work/armv4t-oe-linux-gnueabi/gpsd-3.4-r2/gpsd-3.4/gpsctl.xml:8: warning: failed to load external entity "http://www.oasis-open.org/docbook/xml/4.1.2/docbookx.dtd"
|    "http://www.oasis-open.org/docbook/xml/4.1.2/docbookx.dtd">
|                                                               ^
| I/O error : Attempt to load network entity http://www.oasis-open.org/docbook/xml/4.1.2/docbookx.dtd
| warning: failed to load external entity "http://www.oasis-open.org/docbook/xml/4.1.2/docbookx.dtd"
| validity error : Could not load the external subset "http://www.oasis-open.org/docbook/xml/4.1.2/docbookx.dtd"
| Document /var/lib/jenkins/jobs/shr-core-branches/workspace/shr-core/tmp-eglibc/work/armv4t-oe-linux-gnueabi/gpsd-3.4-r2/gpsd-3.4/gpsctl.xml does not validate
| mv: cannot stat `gpsctl.1': No such file or directory
| scons: *** [gpsctl.1] Error 1
| scons: building terminated because of errors.

Signed-off-by: Martin Jansa <Martin.Jansa@gmail.com>
---
 SConstruct |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/SConstruct b/SConstruct
index 9c0f751..8236abb 100644
--- a/SConstruct
+++ b/SConstruct
@@ -560,12 +560,12 @@ size_t strlcpy(/*@out@*/char *dst, /*@in@*/const char *src, size_t size);
 
 
 manbuilder = mangenerator = htmlbuilder = None
-if config.CheckXsltproc():
+if False and config.CheckXsltproc():
     mangenerator = 'xsltproc'
     build = "xsltproc --nonet %s $SOURCE >$TARGET"
     htmlbuilder = build % docbook_html_uri
     manbuilder = build % docbook_man_uri
-elif WhereIs("xmlto"):
+elif False and WhereIs("xmlto"):
     mangenerator = 'xmlto'
     htmlbuilder = "xmlto html-nochunks $SOURCE; mv `basename $TARGET` $TARGET"
     manbuilder = "xmlto man $SOURCE; mv `basename $TARGET` $TARGET"
-- 
1.7.8.6

